### PR TITLE
Radio button file navigation styling

### PR DESCRIPTION
**Pull request by garethfoote**

### CODE REVIEW

**Comment by garethfoote** [comment]
@MancunianSam Should I change the CSS selectors from `govuk-tna` to `tna` in this so that both components match? I now realise that the `govuk` doesn't get used outside of core GDS stuff.

**Comment by TomJKing** [comment]
> _Should I change the CSS selectors from govuk-tna to tna in this so that both components match? I now realise that the govuk doesn't get used outside of core GDS stuff._

@garethfoote Yes, please.

**Comment by garethfoote** [comment]
@TomJKing @MancunianSam I've changed the class names and made a couple of changes to how we reference classes fom the JS/TS. Ready for review whenever.

File: src/nationalarchives/components/nested-navigation/nested-navigation.unit.test.ts [hunk 1]
```diff
@@ -1,629 +1,696 @@
...
+        parentDiv.append(li);
+        const setFocusToItem = jest.fn();
+
+        const nestedNavigation = createNestedNavigation({ setFocusToItem }, li);
+        nestedNavigation.setFocusToPreviousItem(li);
+
+        expect(setFocusToItem).toHaveBeenCalledWith(siblingLi);
+      });
+
+      it("should go the parent if it exists and there are no siblings", () => {
```
Comment (TomJKing) [inline src/nationalarchives/components/nested-navigation/nested-navigation.unit.test.ts:311]: Maybe reword test slightly: `should go to the parent if ...`

**Review by TomJKing — COMMENTED** [review]
Looks good. One very minor comment. Thank

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks
